### PR TITLE
Update limits_and_triggers.adoc with Cairo step limit and Blockifier …

### PR DESCRIPTION
**Pull request by odednaor**
### Description of the Changes

Change step limit of txs and add a link to the Blockifier constant config file.

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-1199/documentation/tools/limits_and_triggers/

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1199)

### CODE REVIEW

**Review by ArielElp — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1199)* status: all files reviewed, 1 unresolved discussion (waiting on @odednaor and @stoobie)

___
*[`components/Starknet/modules/tools/pages/limits_and_triggers.adoc` line 6 at r1](https://reviewable.io/reviews/starknet-io/starknet-docs/1199#-NtuCsFaEkl02xBtajVa:-NtuCsFaEkl02xBtajVb:b-k9no56) ([raw file](https://github.com/starknet-io/starknet-docs/blob/2eb218fff25ee35d2823900c522806b58851627f/components/Starknet/modules/tools/pages/limits_and_triggers.adoc#L6)):*
> starting from Starknet version v0.13.1

from version 0.13.0 onwards

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 1]
```diff
@@ -24,7 +26,7 @@ The gas limit for Starknet blocks is therefore inherited from the gas limit for
 
 |Max transaction size (Cairo steps)|The maximum number of computational steps, measured in Cairo steps, that a transaction can contain when processed on the Starknet network.
 This limit is important for ensuring the efficient execution of transactions and preventing potential congestion.
-| 3,000,000 | 3,000,000 | 3,000,000
+| 4,000,000 | 4,000,000 | 4,000,000
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:29]: @odednaor Yesterday I was told that the new limits only apply to Sepolia and Mainnet. Are you sure this applies to Goerli as well?

File: components/Starknet/modules/tools/pages/limits_and_triggers.adoc [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 Starknet currently has a number of limits in place in order to keep the network stable and optimized for the best performance.
 
+Blockifier-related constants and limits are defined, starting from Starknet version v0.13.0, in a link:https://github.com/starkware-libs/blockifier/tree/main/crates/blockifier/resources[dedicated file].
```
Comment (stoobie) [inline components/Starknet/modules/tools/pages/limits_and_triggers.adoc:6]:
```suggestion
Blockifier-related constants and limits are defined, starting from Starknet version v0.13.0, in link:https://github.com/starkware-libs/blockifier/blob/main/crates/blockifier/resources/versioned_constants.json[`versioned_constants.json`].
```

**Review by stoobie — COMMENTED** [review]
@odednaor One suggestion, one question.

**Review by stoobie — APPROVED** [review]
@odednaor LGTM. What is the other constants file in the same directory as `versioned_constants.js`?

**Pull request merged**